azkaban-common/src/test/resources/log4j.properties

```diff
@@ -1,4 +1,4 @@
 log4j.rootLogger=INFO, Console
 log4j.appender.Console=org.apache.log4j.ConsoleAppender
 log4j.appender.Console.layout=org.apache.log4j.PatternLayout
-log4j.appender.Console.layout.ConversionPattern=%d{yyyy/MM/dd HH:mm:ss.SSS Z} %p [%c{1}] %m%n
+log4j.appender.Console.layout.ConversionPattern=%d{yyyy/MM/dd HH:mm:ss.SSS Z} %5p [%F:%L] [%t] %m%n
```

Comment:
Just curious, what's the reason for changing `%p` to `%5p`? I *think* it's going to add padding to the severity levels shorter than 5 characters (INFO, WARN). Why do we want it?

Reply:
Yes, this adds padding such that information in log line start from same column(as ERROR, DEBUG are 5 chars) in line number